### PR TITLE
Create Portfolio mobile skeleton

### PR DESCRIPTION
**Pull request by YKerroum**
### Create Homepage:

- Navigation bar.
- Description.

### CODE REVIEW

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @YKerroum , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] Good commit messages :heavy_check_mark: 
- [ ] Good Pull Request description :heavy_check_mark: 
- [ ] Meaningfull branch name :heavy_check_mark: 

## Required Changes ♻️
- [ ] Please kindly add to your root directory the `.hintrc` and `.stylelintrc.json` files. The `Webhint` file helps you improve your site's accessibility, speed, cross-browser compatibility, and more by checking your code for best practices and common errors, and the `Stylelint`  helps you avoid errors and enforce conventions in your styles :wink: :wink: 

- [ ] For other changes please kindly check the comments under the review :point_down: :point_down: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+<body>
+	<div id="container">
+	<header id="box1">
+		<img id="nav-bar1" src="./images/back.svg" alt="background picture">
+		<svg id="nav-bar2" width="20" height="14" viewBox="0 0 20 14" fill="none" xmlns="http://www.w3.org/2000/svg">
+		<path fill-rule="evenodd" clip-rule="evenodd" d="M0 0H20V2H0V0ZM0 6.00002H20V8.00002H0V6.00002ZM20 12H0V14H20V12Z" fill="#1A2236"/>
+		</svg>
+	</header>
+	<section id="box2">
+		<div id="name">Youssef KERROUM</div>
```
[the remark this answers is not in the document]
Comment (Kossi-stack) [reply index.html]: - [ ] you did not fix this

[the remark this answers is not in the document]
Comment (YKerroum) [reply index.html]: @Kossi-stack  Yeah i forgot about it, i fixed it now thanks

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️
**Hi @YKerroum,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 🎉


You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**